Sources/JobsRedis/RedisJobQueue.swift

```diff
@@ -24,31 +25,47 @@ import RediStack
 public final class RedisJobQueue: JobQueueDriver {
     public struct JobID: Sendable, CustomStringConvertible {
         let id: String
+        let delayUntil: Int64
 
-        public init() {
+        public init(delayUntil: Double?) {
```

Comment:
Why don't you hide the Date conversion completely and have the delayUntil value be a Date?
```suggestion
        public init(delayUntil: Date?) {
```